src/core/Toast/Toast.tsx

```diff
@@ -81,14 +87,26 @@ export const Toast = (props: ToastProps) => {
     duration = 7000,
     hasCloseButton,
     onRemove,
+    animateOutTo,
+    placementPosition,
   } = props;
 
   useTheme();
-
   const closeTimeout = React.useRef(0);
 
   const [visible, setVisible] = React.useState(isVisible);
   const [height, setHeight] = React.useState(0);
+  // ref of current toast
+  const thisElement = React.useRef<HTMLDivElement>(null);
+  //variable for margin bottom
```

Comment:
not only for bottom anymore.. I think you can remove this comment, above one too..